src/profile/components/editProfileForm/EditProfileForm.tsx

```diff
@@ -185,7 +185,7 @@ function EditProfileForm(props: Props): React.ReactElement {
       }}
       validationSchema={schema}
     >
-      {(formikProps: FormikProps<FormValues>) => (
+      {(editProfileFormikProps: FormikProps<FormValues>) => (
```

Comment:
This commit (8fb9f7a5ca) doesn't look to me like the right solution. The tool is helpful, it's indicating a code smell -- it says it's code duplication, but it could also be too large components, wrong architecture or a number of other things. Instead of silencing the tool by making worse code, find a solution that **improves** the code. That's anyway the idea in using these tools in the first place.

Reply:
Yes, agreed. I removed the "fix" and the code smell can remain. These components that have duplicate code will be refactored anyway when user data from strong authentication is added. Better to fix this then.